.github/workflows/int-and-connected-test-run.yml

```diff
@@ -108,100 +132,81 @@ jobs:
-          # unset the GitHub Action default JAVA_HOME to run with JDK 17
-          JAVA_HOME:
+          AZURE_SYNAPSE_SQLADMINUSER: ${{ secrets.AZURE_SYNAPSE_SQLADMINUSER }}
+          AZURE_SYNAPSE_SQLADMINPASSWORD: ${{ secrets.AZURE_SYNAPSE_SQLADMINPASSWORD }}
+          AZURE_SYNAPSE_WORKSPACENAME: tdr-snps-int-east-us-ondemand.sql.azuresynapse.net
+          # required for integration tests
+          GOOGLE_APPLICATION_CREDENTIALS: jade-dev-account.json
+          GOOGLE_SA_CERT: jade-dev-account.pem
+          IT_JADE_API_URL: http://localhost:8080
+          # postgres connection details
+          PGHOST: 127.0.0.1
+          PGPASSWORD: postgres
+          # required for integration tests
+          RBS_CLIENT_CREDENTIAL_FILE_PATH: rbs-tools-sa.json
+          RBS_INSTANCEURL: https://buffer.tools.integ.envs.broadinstitute.org
+          RBS_POOLID: datarepo_v1
           # output plain logs instead of json
           TDR_LOG_APPENDER: 'Console-Standard'
-        with:
-          actions_subcommand: 'gradleinttest'
-          pgport: ${{ job.services.postgres.ports[5432] }}
-          test_to_run: 'testIntegration'
-          sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: Clean state lock from used Namespace on API deploy
-        if: always()
-        uses: broadinstitute/datarepo-actions/actions/main@0.74.0
-        with:
-          actions_subcommand: 'k8_checknamespace_clean'
-          sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: Clean whitelisted Runner IP
+        run: |
+          # wait for postgres to be ready
+          pg_isready -h ${PGHOST} -t 30
+          # create the datarepo and stairway databases
+          psql -U postgres -f ./db/create-data-repo-db
+          # extract service account credentials
+          base64 --decode <<< ${{ secrets.SA_B64_CREDENTIALS }} > ${GOOGLE_APPLICATION_CREDENTIALS}
+          jq -r .private_key ${GOOGLE_APPLICATION_CREDENTIALS} > ${GOOGLE_SA_CERT}
+          chmod 644 ${GOOGLE_SA_CERT}
+          base64 --decode <<< ${{ secrets.B64_RBS_APPLICATION_CREDENTIALS }} > ${RBS_CLIENT_CREDENTIAL_FILE_PATH}
+          # build code and run local api
+          ./gradlew bootJar
+          export DATA_REPO_JAR=$(find . -type f -name jade-data-repo-*-SNAPSHOT.jar)
+          java -jar ${DATA_REPO_JAR} > local-api-output.log &
+          # wait until api is ready
+          timeout 30 bash -c 'until curl -s ${IT_JADE_API_URL}/status; do sleep 1; done'
+          # run integration tests
+          ./gradlew --scan testIntegration
+      - name: Upload API logs
         if: always()
-        uses: broadinstitute/datarepo-actions/actions/main@0.74.0
+        uses: actions/upload-artifact@v4
         with:
-          actions_subcommand: 'gcp_whitelist_clean'
-          sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
+          name: api-logs
+          path: local-api-output.log
```

Comment:
It would be helpful to make this name more specific, to indicate that they're logs emitted as part of integration tests in case someone stumbles on them when looking at the test artifacts.

Reply:
Any suggestions for a name? `integration-test-api-logs`, perhaps?

Reply:
Looks good!